internal/config/colorscheme.go

```diff
@@ -96,7 +97,19 @@ func ParseColorscheme(text string) (map[string]tcell.Style, error) {
 			continue
 		}
 
-		matches := parser.FindSubmatch([]byte(line))
+		matches := includeParser.FindSubmatch([]byte(line))
+		if len(matches) == 2 {
+			include := string(matches[1])
+			if include != name {
```

Comment:
So now a colorscheme can include itself. Yet still, what if colorscheme A includes colorscheme B which includes colorscheme A? Or if colorscheme A includes colorscheme B which includes colorscheme C which includes colorscheme D which includes colorscheme E which includes colorscheme A?

Reply:
Yesterday I already thought about that situation and expect that complaint, but didn't came up with a perfect solution so far.
Currently I don't know if it's feasible to allow only one include level.

Reply:
Just a question. Why not to use something like like array/hashmap? If it was already added we just check if it is in the array...?

Reply:
I choose the simple string array...